FlowCrypt/Common/AttesterApi.swift

```diff
@@ -13,24 +13,25 @@ final class AttesterApi {
 
     static let shared: AttesterApi = AttesterApi()
 
-    private let url = "https://flowcrypt.com/attester/"
+    private static let url = "https://flowcrypt.com/attester/"
```

Comment:
I added static so that `lookupEmail` does not have to do the `[weak self]` dance.